apis/python/src/tiledbsoma/options/_soma_tiledb_context.py

```diff
@@ -146,6 +151,15 @@ def timestamp(self) -> Optional[datetime.datetime]:
             return None
         return ms_to_datetime(self.timestamp_ms)
 
+    @property
+    def native_context(self) -> clib.SOMAContext:
+        """The C++ SOMAContext for this SOMA context."""
+        if self._native_context is None:
+            self._native_context = clib.SOMAContext(
+                {k: str(self.tiledb_config[k]) for k in self.tiledb_config}
+            )
+        return self._native_context
+
     @property
     def tiledb_ctx(self) -> tiledb.Ctx:
         """The TileDB Context for this SOMA context."""
```

Comment:
to clarify this comment in light of the `native_context`, maybe rephrase as `The TileDB-Py Context ...`?